tests/test_cbv.py

```diff
@@ -70,27 +120,21 @@ class CBV:
             def g(self) -> bool:
                 return hasattr(self, "class_var")
 
-        client = TestClient(router)
+        return
+
+    def test_class_var(self, cbv_router: None, client: TestClient) -> None:
         response = client.get("/")
-        assert response.status_code == 200
+        assert response.status_code == SUCCESS_CODE
         assert response.content == b"false"
 
-    def test_routes_path_order_preserved(self, router: APIRouter) -> None:
-        @cbv(router)
-        class CBV:
-            @router.get("/test")
-            def get_test(self) -> int:
-                return 1
 
-            @router.get("/{any_path}")
-            def get_any_path(self) -> int:  # Alphabetically before `get_test`
-                return 2
-
-        client = TestClient(router)
-        assert client.get("/test").json() == 1
-        assert client.get("/any_other_path").json() == 2
+class TestMultipleTests(AbstractTestRouter):
+    def setup_method(self):
+        self.item = "abc"
+        self.num_item = "1"

```

Comment:
`num_item` of type `str`?

Reply:
str is for the asser, when getting the custom path you compare 1 to "1" - we can convert it with int(), but do we want to manipulate the response from the router?